Makefile

```diff
@@ -318,6 +324,7 @@ endif
 .PHONY: build-windows
 build-windows:
 	@echo "===> Building Antrea bins and antrea/antrea-windows Docker image <==="
+	docker build --cache-from gavinxin/base-windows:$(WIN_BUILD_TAG) -t antrea/base-windows:$(WIN_BUILD_TAG) -f build/images/base-windows/Dockerfile $(DOCKER_BUILD_ARGS) --network host .
```

Comment:
Is it a tag with 32 chars too long? Maybe it is not necessary to use the complete hash value? What is your point @antoninbas

Reply:
I agree, I would say something like 10 characters is enough

Reply:
Updated